ops/roles/infra/tasks/main.yml

```diff
@@ -45,17 +49,28 @@
     group: '{{ ansible_user }}'
     mode: 0644
 
-- name: Install Chromium
+- name: Install Chromium — Debian
   apt:
     pkg:
       - chromium
     update_cache: yes
     state: latest
+  when: ansible_distribution == 'Debian'
+
+- name: Install Chromium — Ubuntu
+  apt:
+    pkg:
+      - chromium-browser
+    update_cache: yes
+    state: latest
+  when: ansible_distribution == 'Ubuntu'
 
   # See https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#recommended-enable-user-namespace-cloning
 - name: Enable user namespace cloning to allow running Chromium in a sandbox
   command: sysctl -w kernel.unprivileged_userns_clone=1
   when: ansible_facts['architecture'] != "aarch64"
 
 - include_tasks: mongo.yml
-  when: ansible_facts['architecture'] != "aarch64"
+  when:
+    - mongo_required
+    - ansible_facts['architecture'] != "aarch64"
```

Comment:
Can you please add a comment on why it should not run on aarch64.
Also, if this is a mongo version problem, can't we use mongo 4 on ubuntu?

Reply:
Not really, I do not know exactly why we cannot use mongo on aarch64. By the way, it was you that added this condition, so If you remember why, please add the comment yourself :)

Reply:
In order to test, uncomment the lines and try to deploy on vagrant: `ansible-playbook ops/site.yml -i ops/inventories/dev.yml -vvv`